services/web-app/pages/designing/get-started.mdx

```diff
@@ -145,8 +145,6 @@ Everyone using Carbon should use core design kits as a starting point for buildi
 This brings consistency and speed, allowing teams to focus on solving more specific user tasks. Core
 design kits include elements, core components, wireframes, and guidelines.
 
-### Core design kits
-
 <FilterableDesignKitTable
   designKitsData={props.designKitsData}
   designTools={['Figma', 'Sketch', 'Adobe XD', 'Axure']}
```

Comment:
The tool switcher should extend the full length of the table. Check in with Tracey (she was updating the redline)